scripts/build_appimage.sh

```diff
@@ -20,43 +20,119 @@ if [[ $# -lt 2 ]]; then
 	exit 1
 fi
 
-WORKSPACE="$1"
-APPDIR="$2"
+# declare variables
+BUILD_TYPE="${BUILD_TYPE:-RelWithDebInfo}"
+WORKSPACE="$(realpath "$1")"
+APPDIR="$(realpath "$2")"
 WGET_ARGS=(--continue --no-verbose)
 ARCH="$(uname -m)"
 
-# Reset arguments
-set --
-
-# Download linuxdeploy and plugins
-BUNDLER="$WORKSPACE/linuxdeploy.AppImage"
-runCmd wget "${WGET_ARGS[@]}" "https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-${ARCH}.AppImage" --output-document="$BUNDLER" \
-	&& set -- "$@" --output appimage
-runCmd wget "${WGET_ARGS[@]}" "https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-gtk/master/linuxdeploy-plugin-gtk.sh" \
-	&& set -- "$@" --plugin gtk
-runCmd wget "${WGET_ARGS[@]}" "https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-ncurses/master/linuxdeploy-plugin-ncurses.sh" \
-	&& set -- "$@" --plugin ncurses
-runCmd wget "${WGET_ARGS[@]}" "https://raw.githubusercontent.com/linuxdeploy/misc-plugins/master/gettext/linuxdeploy-plugin-gettext.sh" \
-	&& set -- "$@" --plugin gettext
-#if [[ -z "$VERSION" ]]; then
-#	export LINUXDEPLOY_PLUGIN_GDB_SRC="$WORKSPACE/src"
-#	runCmd wget "${WGET_ARGS[@]}" "https://raw.githubusercontent.com/linuxdeploy/misc-plugins/master/gdb/linuxdeploy-plugin-gdb.sh" \
-#		&& set -- "$@" --plugin gdb
-#fi
-runCmd chmod --verbose a+x ./*.AppImage ./*.sh
-
-# Set useful variables for linuxdeploy
+runCmd mkdir -p "$APPDIR"
+
 [[ -n "$VERSION" ]] && export RELEASE="latest" || export RELEASE="continuous"
-export LDAI_UPDATE_INFORMATION="gh-releases-zsync|${GITHUB_REPOSITORY//\//|}|${RELEASE}|CPU-X-*$ARCH.AppImage.zsync"
-export LDAI_VERBOSE=1
-#export DEBUG=1
-export DISABLE_COPYRIGHT_FILES_DEPLOYMENT=1
+export UPDATE_INFORMATION="gh-releases-zsync|${GITHUB_REPOSITORY//\//|}|${RELEASE}|CPU-X-*$ARCH.AppImage.zsync"
+echo "UPDATE_INFORMATION=$UPDATE_INFORMATION"
+
+# Install dependencies
+runCmd pacman -Syu --noconfirm \
+	base-devel \
+	binutils \
+	cairomm \
+	cmake \
+	dconf \
+	findutils \
+	gawk \
+	gcc-libs \
+	git \
+	glib2 \
+	glibc \
+	glibmm \
+	gtkmm3 \
+	libcpuid \
```

Comment:
You can remove libcpuid package here because you build it from sources later.

The Arch package is not always up-to-date (e.g. v0.8.0 is not available yet :disappointed:).